src/devices/tuya.ts

```diff
@@ -1845,8 +1845,11 @@ const definitions: Definition[] = [
         },
     },
     {
-        fingerprint: [{manufacturerName: '_TZ2000_a476raq2'}],
-        zigbeeModel: ['TS0201', 'SNTZ003'],
```

Comment:
Can you use this instead:

```
fingerprint: [{manufacturerName: '_TZ2000_a476raq2'}],
zigbeeModel: ['TS0201', 'SNTZ003', 'TY0201'],
```

Reply:
Yep -- Still works like that but I'll be honest I don't quite understand how given my devices have a different model *and* a different manufacturer name (_TZ3000_bjawzodf)